lang/python27/Portfile

```diff
@@ -68,33 +68,30 @@ openssl.branch      1.1
 
 # This port is used by clang-3.4 to bootstrap libcxx
 subport ${name}-bootstrap {
-    set stdprefix ${prefix}
-    prefix ${prefix}/libexec/libcxx-bootstrap
+    revision                [ expr ${revision} + 1 ]
+
+    set stdprefix           ${prefix}
+    set prefix              ${prefix}/libexec/${subport}
     set frameworks_dir      ${prefix}/Library/Frameworks
     set applications_dir    ${prefix}/Applications
-    configure.cppflags-append   -I${stdprefix}/include
-    configure.ldflags-append    -L${stdprefix}/lib
-    configure.env-append    PATH=${prefix}/bin:$::env(PATH)
+
     configure.cxx_stdlib
     compiler.whitelist      clang llvm-gcc-4.2 gcc-4.2 apple-gcc-4.2
+
     patchfiles-delete       patch-libedit.diff
-    depends_build-replace   port:pkgconfig port:pkgconfig-bootstrap
-    depends_lib-replace     port:ncurses port:ncurses-bootstrap
-    depends_lib-delete      port:db48 \
-                            port:gettext-runtime \
-                            port:libedit \
-                            port:sqlite3
-    # no need to be 'port select'able
+
+    depends_build-delete    port:pkgconfig
+    # OpenSSL PG adds dependency via callback, clena it the same way :)
```

Comment:
Better to avoid including the portgroup in the first place if it shouldn't be used.

Reply:
Good idea, indeed.